packages/client/src/Encrypt.ts

```diff
@@ -15,7 +15,7 @@ export default class PublisherEncryption implements Stoppable {
 
     constructor(
         private streamEndpoints: StreamEndpointsCached,
-        private keyExchange: PublisherKeyExchange,
+        @inject(delay(() => PublisherKeyExchange)) private keyExchange: PublisherKeyExchange,
```

Comment:
Does this fix something related to this PR?

Reply:
Yeah the import paths have changed as a result of these changes, thereby causing circular dependencies.